plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -516,9 +516,9 @@ public void tearDown()
         assertUpdate("DROP TABLE IF EXISTS base_table1");
         assertUpdate("DROP TABLE IF EXISTS base_table2");
         // Drop storage tables
-        MaterializedResult baseResult = computeActual("show tables in tpch");
+        MaterializedResult baseResult = computeActual("select storage_table from system.metadata.materialized_views where catalog_name='iceberg' and schema_name='tpch'");
         for (MaterializedRow row : baseResult.getMaterializedRows()) {
-            assertUpdate("DROP TABLE IF EXISTS " + row.getField(0).toString());
+            assertUpdate("DROP MATERIALIZED VIEW IF EXISTS " + row.getField(0).toString());
```

Comment:
The storage table is a table, so the previous code, using DROP TABLE, seemed correct.
However, IMO this cleanup method is not needed at all --- https://github.com/trinodb/trino/pull/11626

Reply:
@findepi, I can disregard this request in that case. 

This review came from the discussion in https://github.com/trinodb/trino/pull/11555#discussion_r831992696, I am fixing the issue #9129 to raise proper exception. Do you mean the DROP TABLE should exhibit same behavior for materialized views?

cc @findinpath

Reply:
"storage table" is a table, not a materialized view